code/modules/mob/living/carbon/slime/slime.dm

```diff
@@ -83,10 +83,11 @@
 	if(bodytemperature >= 330.23) // 135 F
 		return -1	// slimes become supercharged at high temperatures
 
-	. = ..(1)
+	. = ..()
```

Comment:
realize that this passes the argument to assume `run_speed`, but I did this for consistency with TG and the fact that the intent of `"run"` is default, not to mention future proofing.